app/workers/reset_project_counters_worker.rb

```diff
@@ -0,0 +1,39 @@
+class ResetProjectCountersWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :data_high
+
+  def perform(project_id)
+    project = Project.find(project_id)
+
+    return unless project.launch_date
```

Comment:
what about beta projects, etc that haven't got a launch date? Should we have a default on this field?